### PR TITLE
More fsevent apis

### PR DESCRIPTION
**Pull request by ldm0**

### CODE REVIEW

File: fsevent-sys/src/fsevent.rs [hunk 1]
```diff
@@ -59,43 +65,71 @@ pub const kFSEventStreamEventFlagItemCloned: FSEventStreamEventFlags = 0x0040000
...
 extern "C" {
+    pub fn FSEventStreamCopyDescription(stream_ref: ConstFSEventStreamRef) -> CFStringRef;
+    pub fn FSEventStreamCopyPathsBeingWatched(streamRef: ConstFSEventStreamRef) -> CFArrayRef;
     pub fn FSEventStreamCreate(
-        allocator: cf::CFAllocatorRef,
+        allocator: CFAllocatorRef,
         callback: FSEventStreamCallback,
         context: *const FSEventStreamContext,
-        pathsToWatch: cf::CFMutableArrayRef,
+        pathsToWatch: CFArrayRef,
```
Comment (ldm0) [inline fsevent-sys/src/fsevent.rs:84]: https://developer.apple.com/documentation/coreservices/1443980-fseventstreamcreate

**Comment by ldm0** [comment]
Ping @octplane, this one is much simpler. :-P

**Comment by octplane** [comment]
Thanks, will have a look shortly!

**Comment by octplane** [comment]
Thanks! Now working to do a release :)

**Comment by octplane** [comment]
And everything is released! Thanks again for your time <3
- https://crates.io/crates/fsevent-sys/4.1.0
- https://crates.io/crates/fsevent/2.1.2

**Comment by ldm0** [comment]
> And everything is released! Thanks again for your time <3
> 
> * https://crates.io/crates/fsevent-sys/4.1.0
> * https://crates.io/crates/fsevent/2.1.2

@octplane Thanks! :-)

**Pull request merged**